### DISCUSSION TITLE
[393380]战术小队 Squad

### DISCUSSION THREAD

**Discussion by zhuifengzhe1** [pjy612/SteamManifestCache#633] [Share]
### Steam 游戏ID

`393380`

### 密钥列表

```hash
393381;d11ff0e6b7d56d9c48173256141f75daf1a14f85f6aac9e09d45d05ef7ef935d
1123920;6e595b318ca37cc13710648c00e22386d7c5d385fadef84067f0e22b81b1d468
```

### 清单附件

[6517631680513206435.zip](https://github.com/pjy612/SteamManifestCache/files/15152323/6517631680513206435.zip)

### 备注
Code:39FD614C3C9F9E5A
Email:[asdadad331@outlook.com](mailto:asdadad331@outlook.com)

怎么没人玩战术小队呀，这游戏除了林子大了什么鸟都有和一堆演员开透什么都好，快来玩啊（对了，op没有第一人称观察视野不太好抓挂）
![1714409705904](https://github.com/pjy612/SteamManifestCache/assets/77050692/985e1d96-32b0-4515-95ce-88d1e8094048)

**Comment by pjy612** [comment]
* 秘钥分隔符用错了

**Reply by zhuifengzhe1** [reply]
改好了
